<launch>

  <!-- ROS-Pybullet interface -->
  <node pkg="ros_pybullet_interface" name="ros_pybullet_interface" type="ros_pybullet_interface_node.py" output="screen">
    <rosparam param="config" file="$(find rpbi_examples)/configs/run_franka/config.yaml"/>
  </node>


  <!-- Static transforms -->
  <node pkg="tf2_ros" type="static_transform_publisher" name="floor_tf" args="0 0 0 0 0 0 rpbi/world rpbi/floor"/>




  <!-- Example node that generates target joint positions -->
  <node pkg="rpbi_examples" name="example" type="basic_robot_example_node.py" args="franka" output="screen"/>

</launch>
